File: Lecture2/Calculator.java
package Lecture2;

import java.util.Scanner;

public class Calculator {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        System.out.println("Enter first number:");
        int a = sc.nextInt();
        
        System.out.println("Enter second number:");
        int b = sc.nextInt();
        
        System.out.println("Enter operator (+, -, *, /):");
        char operator = sc.next().charAt(0);
        
        switch (operator) {
            case '+':
                System.out.println("Result: " + (a + b));
                break;
            case '-':
                System.out.println("Result: " + (a - b));
                break;
            case '/':
                if (b != 0) {
                    System.out.println("Result: " + (a / b));
                } else {
                    System.out.println("Cannot divide by zero!");
                }
                break;
            case '*':
                System.out.println("Result: " + (a * b));
                break;
            default:
                System.out.println("Enter valid operator");
                break;
        }
        
        sc.close();
    }
}

File: Lecture2/Calculator.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b;
    char operator_char;
    
    cout << "Enter first number:" << endl;
    cin >> a;
    
    cout << "Enter second number:" << endl;
    cin >> b;
    
    cout << "Enter operator (+, -, *, /):" << endl;
    cin >> operator_char;
    
    switch (operator_char) {
        case '+':
            cout << "Result: " << (a + b) << endl;
            break;
        case '-':
            cout << "Result: " << (a - b) << endl;
            break;
        case '/':
            if (b != 0) {
                cout << "Result: " << (a / b) << endl;
            } else {
                cout << "Cannot divide by zero!" << endl;
            }
            break;
        case '*':
            cout << "Result: " << (a * b) << endl;
            break;
        default:
            cout << "Enter valid operator" << endl;
            break;
    }
    
    return 0;
} 